driver/ppm_fillers.c

```diff
@@ -2982,7 +3004,7 @@ int f_sys_creat_x(struct event_filler_arguments *args)
 	res = val_to_ring(args, open_modes_to_scap(O_CREAT, modes), 0, false, 0);
 	CHECK_RES(res);
 
-	get_fd_dev_ino(retval, &dev, &ino);
+	get_fd_dev_ino_file(retval, &dev, &ino, &file);
```

Comment:
Same question here, why don't we use the same approach of the modern_ebpf? It is seems more intuitive an moreover having a unique logic should be easier to maintain